dockerfiles/evmapp/entrypoint.sh

```diff
@@ -264,6 +270,14 @@ if [ -n "${SCNODE_REST_PASSWORD:-}" ]; then
 fi
 export SCNODE_REST_APIKEYHASH
 
+# set Metrics API password hash
+if [ "${SCNODE_METRICS_ENABLED:-}" = "true" ]; then
```

Comment:
As the default is set above `:-false` there is no need to use `:-` here as there will always be a value in he env var